app/builders/labelling_form_builder.rb

```diff
@@ -0,0 +1,75 @@
+class LabellingFormBuilder < ActionView::Helpers::FormBuilder
+  include ActionView::Helpers::CaptureHelper
+  include ActionView::Helpers::TagHelper
+  include ActionView::Context
+
+  def form_group(attribute, options={}, &block)
+    a = Attribute.new(self, attribute)
+    FormGroup.new(self, a, options).content capture(&block)
+  end
+
+  def text_field_input(attribute, options={}, &block)
+    a = Attribute.new(self, attribute)
+    content = block_given? ? capture(&block) : ""
+    TextField.new(self, a, options).content content
+  end
+
+  # Defaults to "Yes" "No" labels on radio inputs
+  #TODO: make legend do something
+  def radio_button_fieldset(attribute, legend, options={})
+    a = Attribute.new(self, attribute)
+    RadioButtonFieldset.new(self, a, options).content
+  end
```

Comment:
Would be good to have a consistent method names for the API to the builder - eg 
- text_field_input and radio_button_input OR
- text_field and radio_button(s)